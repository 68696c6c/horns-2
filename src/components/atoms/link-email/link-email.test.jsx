/* global describe, it, expect */
import React from 'react'

import { render } from 'setup-test'

import LinkEmail from '.'

describe('LinkEmail', () => {
  it('should render as default', () => {
    const { container } = render(<LinkEmail>example</LinkEmail>)
    expect(container).toMatchSnapshot()
  })
})
