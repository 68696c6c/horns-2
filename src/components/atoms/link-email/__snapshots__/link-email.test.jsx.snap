// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LinkEmail should render as default 1`] = `
<div>
  <a
    class="link-email css-z7mqwi"
    color="background"
    cursor="pointer"
    href="mailto:?"
  >
    example
  </a>
</div>
`;
